contrib/platform/src/com/sun/jna/platform/win32/WinUser.java

```diff
@@ -350,6 +350,31 @@ public HDEVNOTIFY(Pointer p) {
         }
     }
 
+    
+    /**
+     * Contains data to be passed to another application by the WM_COPYDATA message.
+     */
+    public class COPYDATASTRUCT extends Structure {
+
+		public COPYDATASTRUCT() {
+			super();
+		}
+		
+		public COPYDATASTRUCT(Pointer p) {
+			super(p);
+			//Receiving data and read it from native memory to fill the structure.
+			read();
+		}
+		
+		public ULONG_PTR dwData;
+		public DWORD cbData;
```

Comment:
This could be mapped to int instead (DWORD and int both reference 32 bit wide data), that makes it easier to deal with it.